vaccine_feed_ingest/runners/wa/prepmod/parse.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+import json
+import pathlib
+import sys
+
+from bs4 import BeautifulSoup
+
+input_dir = pathlib.Path(sys.argv[2])
+output_dir = pathlib.Path(sys.argv[1])
+output_file = output_dir / "data.parsed.ndjson"
+
+input_filenames = [p for p in pathlib.Path(input_dir).iterdir() if p.is_file()]
+
+
+def find_data_item(parent, label, offset):
+    try:
+        row_matches = [
+            x for x in parent.find_all(["p", "div"]) if label in x.get_text()
+        ]
+        content = row_matches[-1].contents[offset]
+        return (
+            content.strip() if isinstance(content, str) else content.get_text().strip()
+        )
+    except Exception:
+        return ""
+
+
+with output_file.open("w") as fout:
+    for filename in input_filenames:
+        text = open(filename, "r").read()
+        soup = BeautifulSoup(text, "html.parser")
+
+        # classes only used on titles for search results
+        for title in soup.select(".text-xl.font-black"):
+            parent = title.parent
+            name = title.get_text().strip()
```

Comment:
Looks good. If possible, it'd be great if you could parse out the date of the clinic from the rest of the name. That should lighten the load a bit during the normalization step. Otherwise, ready for merge :)